File: Java Data/T 01 Patterns/Pattern_05_Inverted_Right_Pyramid.java
import java.util.Scanner;

// * * * * *
// * * * *
// * * *
// * *
// *

public class Pattern_05_Inverted_Right_Pyramid {

    public static void invertedRightPyramidPattern(int n) {
        // Outer loop controls the rows. It runs 'n' times (once for each row)
        for (int i = 1; i <= n; i++) {
            // Inner loop for columns, printing stars based on the remaining number of rows
            for (int j = 1; j <= n - i + 1; j++) {
                System.out.print("* ");
            }
            // Move to the next line after each row is complete
            System.out.println();
        }
    }

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of rows for the inverted right pyramid pattern: ");
        int n = scanner.nextInt();

        invertedRightPyramidPattern(n);

        scanner.close();
    }
}

File: 00-Patterns-Python/Pattern_05_Inverted_Right_Pyramid.py
# * * * * *
# * * * *
# * * *
# * *
# *

def inverted_right_pyramid_pattern(n):

    # Outer loop controls the rows. It runs 'n' times (once for each row)
    for i in range(1, n + 1, 1):

        # Inner loop controls the columns (numbers printed in the current row
        for j in range(1, n - i + 2, 1):
            print("* ", end="")

        # Move to the next line after each row is complete
        print()

num = input("Enter the number of rows for the inverted right pyramid pattern: ")
inverted_right_pyramid_pattern(int(num))
